source/deploy/system-deploy-interface.rst

```diff
@@ -0,0 +1,148 @@
+Interface between the deploy system and projects
+================================================
+
+This interface isn't always as clear-cut as it could be. For now, we'll
+try to document it as-is.
+
+Definitions
+~~~~~~~~~~~
+
+`Project`: One or more sites that share a code base. In other words,
+if sites' Django code comes from the same source control repository,
+we'll call them part of the same project.  The Twelve-Factor App `calls
+this the app <http://12factor.net/codebase>`_.
```

Comment:
not familiar with "The Twelve-Factor App". What is it? Do we base any of our work on this? If we're going to use it to back up what we're doing, should we introduce it to those who don't know it?